presto-main/src/main/java/com/facebook/presto/sql/planner/QueryPlanner.java

```diff
@@ -557,7 +555,7 @@ private PlanBuilder window(PlanBuilder subPlan, QuerySpecification node)
                 frameEndSymbol = Optional.of(subPlan.translate(frameEnd));
             }
 
-            WindowNode.Frame frame = new WindowNode.Frame(frameType,
+            WindowNode.Frame rewrittenFrame = new WindowNode.Frame(frameType,
                     frameStartType, frameStartSymbol,
                     frameEndType, frameEndSymbol);
```

Comment:
Is it really equivalent to what was before?
When there's no `end` given for a frame, then https://github.com/Teradata/presto/pull/304/commits/f5a57febdbef7b62f1ef628aaede4c40d666c32b#diff-21d31cb9d487772413eff4f67b8555deL516 does not set `frameEndType`, nor `frameEnd` which falls back to default of https://github.com/Teradata/presto/pull/304/commits/f5a57febdbef7b62f1ef628aaede4c40d666c32b#diff-21d31cb9d487772413eff4f67b8555deL505.
On this patch if we won't enter https://github.com/Teradata/presto/pull/304/commits/f5a57febdbef7b62f1ef628aaede4c40d666c32b#diff-21d31cb9d487772413eff4f67b8555deR515 we end up setting default value - `null` (https://github.com/Teradata/presto/pull/304/commits/f5a57febdbef7b62f1ef628aaede4c40d666c32b#diff-21d31cb9d487772413eff4f67b8555deR505).
Is that correct behavior?